qiskit/backends/_qeremote.py

```diff
@@ -0,0 +1,112 @@
+import time
+import logging
+from qiskit.backends._basebackend import BaseBackend
+from qiskit.backends._backendutils import get_backend_configuration
+from qiskit import _openquantumcompiler as openquantumcompiler
+from qiskit._result import Result
+from IBMQuantumExperience.IBMQuantumExperience import IBMQuantumExperience
```

Comment:
Could you please add some missing imports: `ResultError`, `QISKitError`, and `pprint`?

Reply:
Thanks for catching these!